index.js

```diff
@@ -1 +1,89 @@
-// Do work!
\ No newline at end of file
+/* eslint-disable indent */
+// // Make a password function
+// function validatePassword(password) {
+// let status = []
+
+//     if () {
+//         let status = true;
+//     }
+
+
+
+// // Success! 
+
+// // Not long enough, needs >= 8 chars
+//     if (password.length < 8) {
+//         let status = false;
+//     }
+// // Missing a lower case letter
+//     if (password.case) {
+
+//     }
+// // Missing an upper case letter
+
+// // Missing a numeric value
+
+// // Missing a special character
+
+// }
+
+
+
+
+
+function validatePassword(password) {
+    const validLength = (password.length >= 8)
+
+    const validChars = pushCharacters(password)
+
+    if (validLength && validChars) {
+        return true
+    }
+
+    return false
+}
+
+function pushCharacters(password) {
+    let specChars = '`~!@#$%^&*()_+'
+
+    let charStatus = {
```

Comment:
This use of an object to track the checks that have to pass is a great thing to see here.  All of these things are grouped together. You did something original here that is a good thing to see. Nice and organized.